report_paper_wkhtmltopdf_params/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===============================
+Report Paper Wkhtmltopdf Params
+===============================
+
+This module allows you to add new parameters for a paper format which are
+then forwarded to wkhtmltopdf command as arguments.
+
+Usage
+=====
+
+* Go to *Settings* and press 'Activate the developer mode (with assets)'
+
+* Go to *Settings - Technical - Reports - Paper Format*
+
+* Change the parameter 'Disable Smart Shrinking'.
```

Comment:
Why smart shrinking is so special? (I know: it fitted your needs)
My point is why having a module for this special attribute and allow then to have another free text for whatever attributes you want to add.
I would rather have it without the checkbox for smart-shrinking (then you just have to set up the module for your own usage) or with all options available for any (or most of) the good options that wkhtml2pdf can offer.
Option 1 is more flexible and lighter and more prone to errors
Option 2 avoids Odoo crashing controlling the input (which somehow I kinda prefer)

What do you think?